apps/io-services-cms-webapp/src/watchers/on-service-lifecycle-change.ts

```diff
@@ -35,29 +35,34 @@ type RequestHistoricizationAction = Action<
   Queue.RequestHistoricizationItem
 >;
 
-const onAnyChangesHandler = (
-  item: ServiceLifecycle.ItemType
-): RequestHistoricizationAction => ({
+const onAnyChangesHandler = ({
+  _ts,
+  _etag,
+  ...otherProps
+}: ServiceLifecycle.CosmosResource): RequestHistoricizationAction => ({
   requestHistoricization: {
-    ...item,
-    last_update:
-      item.last_update ?? (new Date().toISOString() as NonEmptyString), // last_update fallback (value is always set by persistence layer) TODO add log
+    ...otherProps,
+    // eslint-disable-next-line no-underscore-dangle
+    last_update: new Date(_ts * 1000).toISOString() as NonEmptyString, // last_update on service-history record corresponds to the _ts on the service-lifecycle record
+    // eslint-disable-next-line no-underscore-dangle
+    version: _etag as NonEmptyString, // version on service-history record corresponds to the _etag on the service-lifecycle record
   },
 });
 
 const onSubmitHandler = (
-  item: ServiceLifecycle.ItemType
+  item: ServiceLifecycle.CosmosResource
 ): RequestReviewAction => ({
   requestReview: {
     id: item.id,
     data: item.data,
-    version: item.version ?? (`ERR_${ulidGenerator()}` as NonEmptyString), // TODO add log
+    // eslint-disable-next-line no-underscore-dangle
+    version: item._etag ?? (`ERR_${ulidGenerator()}` as NonEmptyString), // TODO add log
```

Comment:
**chore (non-blocking):** remove useless logic.

Now, using `_etag` field, this logic is no more useful (it will always be valued)

cc: @giuseppedipinto I'm tagging you for a double check, since you added this logic at the time 😉

Reply:
e7143dee0696d567cb39874ff987014daf0a275f